worlds/witness/options.py

```diff
@@ -123,26 +143,33 @@ class EnvironmentalPuzzlesDifficulty(Choice):
 class ObeliskKeys(DefaultOnToggle):
     """
     Add one Obelisk Key item per Obelisk, locking you out of solving any of the associated Environmental Puzzles.
+
     Does nothing if "Shuffle Environmental Puzzles" is set to "off".
     """
     display_name = "Obelisk Keys"
 
 
 class ShufflePostgame(Toggle):
-    """Adds locations into the pool that are guaranteed to become accessible after or at the same time as your goal.
-    Use this if you don't play with release on victory. IMPORTANT NOTE: The possibility of your second
-    "Progressive Dots" showing up in the Caves is ignored, they will still be considered "postgame" in base settings."""
+    """
+    Adds locations into the pool that are guaranteed to become accessible after or at the same time as your goal.
+    Use this if you don't play with release on victory.
+
+    IMPORTANT NOTE: The possibility of your second "Progressive Dots" showing up in the Caves is ignored, they will still be considered "postgame" in base settings.

```

Comment:
I wonder if this note can be removed, since there's honestly a few other things being accounted for. Did people ask about this before it was added to the tooltip?

Reply:
That's probably true, it's one of those things you probably wouldn't have ever considered if the option tooltip didn't point it out to you, I can probably just remove that